packages/cos/collection.yaml

```diff
@@ -2,7 +2,7 @@ packages:
   - &cos
     name: "cos"
     category: "system"
-    version: 0.8.9-18
+    version: 0.8.9-19
```

Comment:
ouch, looks like the rebase merged the bump too, so now that's not being picked up by CI

Reply:
I was thinking that we want to increment this version? how should I fix it then ?